cli/import_map.ts

```diff
@@ -26,8 +28,17 @@ export async function importMap(url: URL) {
     config.importMap,
     JSON.stringify(importMap, null, 2) + "\n",
   );
-  await Deno.writeTextFile("deno.json", JSON.stringify(config, null, 2) + "\n");
 
-  console.log(brightGreen("Deno configuration file saved:"), "deno.json");
+  if (await exists("deno.jsonc")) {
+    promptConfigUpdate({ importMap: config.importMap, tasks: config.tasks });
```

Comment:
It would be nice to show the `deno.jsonc` warning only if there's something to change. Most of the time, the `upgrade` or `import-map` commands only update the `import_map.json` file.